### ISSUE TITLE
Crash upon loading singleplayer world

### ISSUE THREAD

**Issue by RisingStorm71** [GTNewHorizons/GT-New-Horizons-Modpack#6284]
#### Which modpack version are you using?
2.0.9.0QF4
#
#### If in multiplayer; On which server does this happen? 
N/A
#
#### What did you try to do, and what did you expect to happen?
Loading up a single player world I had played in once before. 
#
#### What happened instead? (Attach screenshots if needed)
As soon as the world loaded the game crashed. 
[crash-2020-06-15_14.47.37-server.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/4782394/crash-2020-06-15_14.47.37-server.txt)

I did have Optifine installed with Fastcraft disabled, no other changes were made.

#

**Comment by RisingStorm71** [comment]
I am also on MultiMC if that matters

**Comment by chochem** [comment]
somebody else posted this log earlier today on discord: https://paste.ee/p/Eo1fe
seems to be a common problem.

**Comment by RisingStorm71** [comment]
I have been able to access the world since then after reloading the pack a couple of times, without having to restore from a backup. I didn't change anything as far as I know, so I am not sure what the underlying issue is. It seems to be random.

**Comment by bartimaeusnek** [comment]
@RisingStorm71 for this we would need a full log...

**Comment by ben0sharples** [comment]
https://paste.ee/p/8Ss3b

Here's my full log from when this happened to me if that helps. If there's anything more useful to provide let me know.

I also have Optifine installed, though I upgraded FastCraft instead of disabling it (both options are fine to fix the tessellation-related incompatibility as far as I'm aware).

This error occurs... about 50% of the time upon loading a previous world for me. The crash occurs immediately afters the world finishes loading. That is, just after the "Downloading terrain..." screen disappears, there is a very brief flash of the loaded world, then a crash to desktop.

I'm trying to think of any other information I can provide but that really is all of the reproduction steps - just trying to load any world. Happy to provide anything else that might help though. Cheers.

**Comment by ben0sharples** [comment]
I've found what I _think_ is causing the issue - the crash occurs when you alt-tab during the loading of a world. Keeping focus on the game throughout loading seems to let it load OK.

I guess there's some change in the way Windows is running the game while unfocused that's causing issues in that particular time, though I'm not nearly knowledgeable enough to know what that could be.

If there's any other information I can give to help fix this issue then let me know, though obviously the game is playable again now I've found this.

**Comment by glowredman** [comment]
Not doing anything while the pack/world loads has always been good practice for me and when the world loaded only look around until the FPS have settled.

**Comment by Prometheus0000** [comment]
Whereas I always alt-tab out until the game loads, both at the initial screen, and when loading a world. No problems.

**Comment by Zeruel13** [comment]
Has anyone found the reason for this and/or a solution? I've been getting the ticking memory connection crash upon world loading very often.

**Comment by mitchej123** [comment]
The full logs are expired, need a new one, with a stacktrace.

**Comment by ben0sharples** [comment]
I was previously letting GT:NH load up while I was playing another game due to the long load time. I stopped doing this (i.e. closing as many other applications as possible before starting up the pack) and haven't had the crash since. Not sure if this is "the solution" but I'm presuming it's something to do with memory usage. Not really the pack's fault, just me overestimating my PC.

It would also line up with the Alt-Tabbing explanation, even though I found that that wasn't the cause (had instances both of crashing without Alt-Tabbing, and Alt-Tabbing without crashing). Presumably it was another game that I was Alt-Tabbing to that would be causing the issue, and times when I was wasn't Alt-Tabbing were times when I didn't have anything to Alt-Tab to.

Sorry I can't provide another crash log as I haven't had the issue in a while now. Hopefully this helps for Zeruel in some way.
